core/src/validator/src/rules/key-white-space.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseValidator } from "../core.ts";
+
+import { getStates } from "../utils/getState.ts";
+
+import type { Result } from "../utils/types.ts";
+import { metadataValidator } from "../utils/metadataChecks.ts";
+import { findWhitespace } from "../utils/whiteSpace.ts";
+
+export class KeyWhiteSpace extends BaseValidator {
+  constructor() {
+    const id = "key-white-space";
+    super(id);
+  }
+
+  async Execute(
```

Comment:
this.Logic is synchronous so Execute should be synchronous and shouldnt return a promise

Reply:
removed